### ISSUE TITLE
Change IS and AP text area to text field

### ISSUE THREAD

**Issue by ksherb1** [Cyber4All/bloomin-onion-client#5]
The Onion app should be oriented more towards curriculum proposal than towards implementation, which should happen in the Neutrino project. Instructional strategies and assessment plans should have text annotations, but they needn't do much more than distinguish between other strategies/plans with the same class.

**Comment by DonSeannelly** [comment]
@ksherb1 I believe that Dr. Kaza is looking for us to allow rich text pasting into a text area for this. Can you discuss with him and report back the final decision here?

**Comment by DonSeannelly** [comment]
If we can find a library that will support pasting of rich text into the text areas then we should keep them in. Otherwise, they should be taken out until we decide the best flow of content submission.

**Comment by tylernhoward** [comment]
There are a few open-source options. 

I like this one.. :)
Going to dive into the docs a little more and see how bad this would be to implement quickly
https://github.com/quilljs/quill/

**Comment by tylernhoward** [comment]
https://github.com/iDoRecall/comparisons/blob/master/JavaScript-WYSIWYG-editors.md
Some more light reading, will try  to get one up and running tonight

**Comment by tylernhoward** [comment]
The current version is using CKEditor (Yes, I switched editors 3  times). I think this is the most mature and complete editor. I removed some of the more in-depth plugins that I thought would be unnecessary for our use. 

Here is the documentation for the library: https://docs.ckeditor.com/ckeditor4/docs/
Here is the module we are using to interact with angular: https://github.com/chymz/ng2-ckeditor

This is all residing in a new component called TextEditorComponent, and is able to capture a string in the form of the source output.. For instance, if I pasted basic text, with no formatting it would be "stored" as <p>some text</p> making it easy to save these inputs when able to.
